src/handlers/legacy_redirect_handler.cr

```diff
@@ -12,6 +12,7 @@ class LegacyRedirectHandler
     "/guides/custom-queries"                      => Guides::Database::RawSql,
     "/guides/saving-with-forms"                   => Guides::Database::ValidatingSaving,
     "/guides/asset-handling"                      => Guides::Frontend::AssetHandling,
+    "/guides/internationalization"                => Guides::Frontend::Internationalization,
```

Comment:
Since this is a new page, it doesn't need a legacy redirect. New pages go in `src/models/guides_list.cr`. This one should go last in the FrontEnd category

Reply:
ah yup. now it does what I expect